annotations/shared/src/main/scala/caseapp/Annotations.scala

```diff
@@ -44,17 +44,8 @@ final case class AppVersion(appVersion: String) extends StaticAnnotation
 final case class ArgsName(argsName: String) extends StaticAnnotation
 
 /** Don't parse the annotated field as a single argument. Recurse on its fields instead.
-  *
-  * Optionally, add a prefix to all the names of the fields.
   */
-final case class Recurse(prefix: String) extends StaticAnnotation {
-  def this() = this("")
-}
-
-object Recurse {
-  def apply(): Recurse =
-    new Recurse
-}
+final class Recurse extends StaticAnnotation
```

Comment:
this should not have been reverted

Reply:
hm but then the mimi failure is correct: This is not compatible with the snapshot that does contain this